assets/chat/js/chat.js

```diff
@@ -1294,7 +1191,9 @@ class Chat {
         );
         break;
       default:
-        message = MessageBuilder.error(errorstrings.get(desc) || desc);
+        message = MessageBuilder.error(
+          errorstrings[desc.replace(/'/g, '')] || desc
```

Comment:
Can you explain why this is necessary?

Reply:
Think I missed my weetabix that day😆I was seeing phantom quotes.